apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -159,6 +160,111 @@ function CopyScheduleButton() {
     );
 }
 
+interface CustomEventsBoxProps {
+    customEventsFromProps: RepeatingCustomEvent[];
+    isSkeletonMode: boolean;
+    scheduleNames: string[];
+}
+
+function CustomEventsBox(props: CustomEventsBoxProps) {
+    const { customEventsFromProps, scheduleNames, isSkeletonMode } = props;
+
+    const [customEvents, setCustomEvents] = useState(customEventsFromProps);
```

Comment:
Here I've used a bit of an odd prop, `customEventsFromProps` for two reasons.

1. I can't use customEvents twice (for the prop and the useState)
2. skeletonSchedule doesn't have the luxury of calling the AppStore

I'm open to refactoring the name or the code here if it's too confusing or non-idiomatic